flask_appbuilder/tests/test_api.py

```diff
@@ -2011,14 +2011,23 @@ def test_update_item_excluded_cols(self):
             .one_or_none()
         )
         pk = model1.id
-        item = dict(field_string="test_Put", field_integer=1000)
+        item = dict(field_string="test_Put")
```

Comment:
Breaking change, we are strict now, unknown columns will get `{"message": {"field_integer": ["Unknown field."]}}`